contracts/zero-ex/test/features/native_orders_feature_test.ts

```diff
@@ -1569,4 +1595,310 @@ blockchainTests.resets('NativeOrdersFeature', env => {
             }
         });
     });
+
+    describe.only('fillTakerSignedRfqOrder', () => {
+        it('gas cost if if there is no balance is fully sold, new tokens after trade for both parties', async () => {
```

Comment:
Can we convert these "gas cost" tests into real unit tests?

Reply:
I would also prefer to delete any tests that are just gas cost tests since ganache is a total lie anyway.

Reply:
these tests are not finalized, I just wanted to get a rough estimate of relative costs, if we move forward I'll change these into useful tests